### ISSUE TITLE
EMD/Chamfer Loss doubt

### ISSUE THREAD

**Issue by rjtshrm** [yulequan/PU-Net#29]
Hello, It might be a silly doubt but just curious to know 

- why you are dividing the loss function (EMD or Chamfer) with radius (furthest distance) https://github.com/yulequan/PU-Net/blob/d6f06d8e550b34f6e66db4f69d71854b5aea74d4/code/model_utils.py#L46. Does this has any effect in overall training of data?

- why dist forward and backward are multiplied by weights 0.8 & 0.2. I have referenced other papers as well they just take its as sum without any weight https://github.com/yulequan/PU-Net/blob/d6f06d8e550b34f6e66db4f69d71854b5aea74d4/code/model_utils.py#L56

**Comment by yulequan** [comment]
1. At the beginning, we normalize the distance to stabilize the learning. However, we have already normalized the input, so it has little effect on the training processing.  
2. This is borrowed from the point set generalization paper. In my view, the different weights make the predicted points to be closer to the ground truth points.
